### PR TITLE
Fix: remove loading file section, fix dark mode

### PR DESCRIPTION
**Pull request by StrongMonkey**
- Drop `loading file` section
- Fix the knowledge details page in dark mode

### CODE REVIEW

**Review by ryanhopperlowe — CHANGES_REQUESTED** [review]
sorry to do this, but we can't use tailwind builtin colors. Please only use colors listed in `tailwind.config.js` and `tailwind.css`

File: ui/admin/app/components/knowledge/KnowledgeSourceDetail.tsx [hunk 1]
```diff
@@ -313,14 +313,14 @@ const KnowledgeSourceDetail: FC<KnowledgeSourceDetailProps> = ({
 
     return (
         <Dialog open={isOpen} onOpenChange={onOpenChange}>
-            <DialogContent className=" h-[80vh] w-[80vw] max-w-none max-h-none">
+            <DialogContent className="h-[80vh] w-[80vw] max-w-none max-h-none dark:bg-gray-900">
                 <div
-                    className=" sticky top-0 bg-white z-10 overflow-y-auto mr-2"
+                    className="sticky top-0 bg-white dark:bg-gray-900 z-10 overflow-y-auto mr-2"
```
Comment (ryanhopperlowe) [inline ui/admin/app/components/knowledge/KnowledgeSourceDetail.tsx]: I'm having trouble locating this component in the UI , but my blind guess is you could replace this entire PR by removing `bg-white` from this line

**Comment by StrongMonkey** [comment]
yeah @ryanhopperlowe you are right, I was using some inner div which set bd to white. I removed that and everything seems to work fine. Changed to remove the extra div.

File: ui/admin/app/components/knowledge/KnowledgeSourceDetail.tsx [hunk 2]
```diff
@@ -313,760 +313,723 @@ const KnowledgeSourceDetail: FC<KnowledgeSourceDetailProps> = ({
 
     return (
         <Dialog open={isOpen} onOpenChange={onOpenChange}>
-            <DialogContent className=" h-[80vh] w-[80vw] max-w-none max-h-none">
-                <div
```
Comment (StrongMonkey) [inline ui/admin/app/components/knowledge/KnowledgeSourceDetail.tsx:317]: The only change is the div className moved to DialogContent.

**Review by ryanhopperlowe — APPROVED** [review]

**Comment by ryanhopperlowe** [comment]
> yeah @ryanhopperlowe you are right, I was using some inner div which set bd to white. I removed that and everything seems to work fine. Changed to remove the extra div.

Sweet! thanks for fixing it!

File: ui/admin/app/components/knowledge/KnowledgeSourceDetail.tsx (hunk 1 above)
Comment (StrongMonkey) [reply ui/admin/app/components/knowledge/KnowledgeSourceDetail.tsx]: yeah, this is side effect of using too much ai 😄

**Pull request merged**